libcontainer/cgroups/utils.go

```diff
@@ -399,16 +403,19 @@ func WriteCgroupProc(dir string, pid int) error {
 	return err
 }
 
-// Since the OCI spec is designed for cgroup v1, in some cases
-// there is need to convert from the cgroup v1 configuration to cgroup v2
-// the formula for cpuShares is y = (1 + ((x - 2) * 9999) / 262142)
+// ConvertCPUSharesToCgroupV2Value converts a CPU shares cgroup v1 configuration
+// to cgroup v2.
+//
+// The OCI spec is designed for cgroup v1, in some cases there is need to
+// convert from the cgroup v1 configuration to cgroup v2 the formula for cpuShares
+// is y = (1 + ((x - 2) * 9999) / 262142)
 // convert from [2-262144] to [1-10000]
 // 262144 comes from Linux kernel definition "#define MAX_SHARES (1UL << 18)"
```

Comment:
This line needs to be removed as this is already explained earlier.

And again, the way it was written makes more sense -- we explain
that we convert from the range [A-B] to [C-D], and then explan
where the B comes from.

After the commit, the explanation for B is separated out, which
makes it harder to understand.